browser/components/TodolistPercentage.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @fileoverview Percentage of todo achievement.
+ */
+
+import React, { PropTypes } from 'react'
+import CSSModules from 'browser/lib/CSSModules'
+import styles from './TodoListPercentage.styl'
+
+/**
+ * @param {number} percentageOfTodo
+ */
+
+const TodoListPercentage = ({
+  percentageOfTodo
+}) => (
+  <div styleName='percentageBar' style={{display: isNaN(percentageOfTodo) ? 'none' : ''}}>
+    <div styleName='progressBar' style={{width: percentageOfTodo + '%'}}>
+      <p styleName='percentageText'>{percentageOfTodo + '%'}</p>
```

Comment:
Thank you for telling me. Oh...  I had overlooked `+` and `*`. I'll add these operator.